src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -73,8 +75,13 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
 
     val workflowRecs: Seq[WorkflowRecord] = setWorkflowBatchToQueued(workflowSubmission.batchSize, testData.submission1.submissionId)
 
-    assertResult(SubmitWorkflowBatch(workflowRecs.map(_.id))) {
-      Await.result(workflowSubmission.getUnlaunchedWorkflowBatch(), Duration.Inf)
+    val workflowSubMsg = Await.result(workflowSubmission.getUnlaunchedWorkflowBatch(), Duration.Inf)
+    workflowSubMsg match {
+      case SubmitWorkflowBatch(workflowIds) =>
+        assertResult( workflowRecs.map(_.id).toSet ) {
```

Comment:
Can use `workflowIds should contain theSameElementsAs workflowRecs.map(_.id)` instead